pkgs/by-name/lo/localstack/package.nix

```
@@ -47,13 +50,22 @@ python3.pkgs.buildPythonApplication rec {
 
   pythonImportsCheck = [ "localstack" ];
 
+  # Localstack uses a framework called Plux to do all kinds of dynamic code loading.
+  # Due to this, we'll encounter build errors when it attempts to set up cache directories
+  # in the sandboxed build environment unless we tell it to create it's cache in a safe
+  # directory.
+  makeWrapperArgs = [
+    "--set-default"
+    "XDG_CACHE_HOME"
+    "/tmp/cache"
+  ];
```

Is this needed at runtime (outside of testing the package)? Otherwise it might be better to use this instead:
```suggestion
  nativeCheckInputs = [ writableTmpDirAsHomeHook ];
```